### PR TITLE
Adds note about minimal minishift version

### PR DESCRIPTION
**Pull request by lordofthejars**

### CODE REVIEW

File: documentation/modules/ROOT/pages/1setup.adoc [hunk 1]
```diff
@@ -33,7 +33,7 @@ You will need in this tutorial:
 [#minishift]
 == Setup minishift
 
-Assumes `minishift`, tested with minishift v1.22.0+7163416
+In case of using Minishift you need at least minishift v1.24.0 or above.
```
Comment (kameshsampath) [inline documentation/modules/ROOT/pages/1setup.adoc]: can you rephrase:

`In case of using Minishift, you need to be at least on minishift v1.24.0`  ( just removed or above)

**Comment by lordofthejars** [comment]
@kameshsampath Done.

**Comment by kameshsampath** [comment]
Thanks @lordofthejars  merged
